stories/src/app/components/StoryBox.tsx

```diff
@@ -0,0 +1,17 @@
+interface Props {
+    storyName: string,
```

Comment:
since this is for a story, what about just 'name' and 'description' for these fields?

Reply:
Done!